akvo/rsr/usecases/change_project_parent.py

```diff
@@ -105,6 +109,12 @@ def get_rf_change_candidates(project, new_parent):
 
 
 def change_parent(project, new_parent, verbosity=0):
+    """
+    This function changes a project's parent including its Result Framework objects by traversing up the hierarchy
```

Comment:
Usually python docstrings have a short summary line followed by a longer description, similar to a git commit message. I'll update this docstring and update the commit.